### PR TITLE
Fix amp validator

### PR DESCRIPTION
**Pull request by HarryVerhoef**
**Overall change:**
Prevents git push if amp validation fails, ensures that if amp validation fails then the test suite fails.

**Code changes:**

- Excludes an asset that has a page-weight that is too large.
- Process exits with exit code = 1 if validation fails on any asset.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

File: scripts/ampHtmlValidator/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const pageTypes = [
 // list of urls we have decided are acceptable to fail amp validation
 const excludedUrls = [
   '/mundo/23263889' /* https://github.com/bbc/simorgh/issues/8104 */,
+  '/mundo/noticias-internacional-51266689',
```
Comment (andrewscfc) [inline scripts/ampHtmlValidator/index.js:26]: Why is this url excluded?

**Review by andrewscfc — COMMENTED** [review]
Thanks for getting on this so quickly, just one question

File: scripts/ampHtmlValidator/index.js (hunk 1 above)
Comment (HarryVerhoef) [reply scripts/ampHtmlValidator/index.js:26]: The asset is too large so it fails the validation:

`/mundo/noticias-internacional-51266689
FAIL
line 1, col 5935: The author stylesheet specified in tag 'style amp-custom' is too long - document contains 76818 bytes whereas the limit is 75000 bytes. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#maximum-size)`

**Review by HarveyPeachey — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

File: scripts/ampHtmlValidator/index.js (hunk 1 above)
Comment (andrewscfc) [reply scripts/ampHtmlValidator/index.js:26]: hmm, the other asset that is excluded is a bit unrealistic but this one isn't too bad in terms of realism, might be worth checking with @neildoughty / @gavinspence the number of valid to invalid assets just to check things aren't slowly getting worse.

We made the decision to exclude the other asset here: https://github.com/bbc/simorgh/issues/8104#issuecomment-756616723

**Review by andrewscfc — APPROVED** [review]

File: scripts/ampHtmlValidator/index.js (hunk 1 above)
Comment (andrewscfc) [reply scripts/ampHtmlValidator/index.js:26]: Please don't block the merge of this on the above 👍

**Pull request merged**